content/en/Getting started/glossary.md

```diff
@@ -75,6 +75,65 @@ refines best practice security solutions.
 One of the test criteria used by our pentesters is [CIS Controls v8](https://www.cisecurity.org/controls/v8),
 released in 2021.
 
+## Cobalt Users
+
+When using the Cobalt UI, you may encounter a variety of different users, in the following
+categories:
+
+- Organization Roles: If you're a Cobalt customer, your account may have one or more of the following roles:
+  - Organization Owner: Also known as an _Org Owner_
+  - Organization Member: Also known as an _Org Member_
+  - Pentest Team Member: Also known as a _PT Team Member_
+
+- Pentester Roles: If you're a Cobalt pentester, you may be a _PT Pentester_ or a _PT Lead_.
+
+Select Cobalt employees may be assigned as administrators, as _Cobalt Staff_.
+
+You can review a list of permissions associated with each organization role in the following
+article: [What do the user roles mean?](https://cobaltio.zendesk.com/hc/en-us/articles/360057093472-What-do-the-user-roles-mean-).
+
+<!-- Per https://zombie.atlassian.net/browse/DOCS-5 I should add info from an internal
+spreadsheet, but need to find pull code in Hugo -->
+
+### Organization Member
+
+An _Org Member_ is a user of the Cobalt App who can create an [asset](#asset) or a
+[pentest](#pentest). That user can also see the pentesters who are working on their asset.
+If allowed by their {{% ptaas-tier %}}, they can also manage integration with Jira and GitHub.
+
+### Organization Owner
+
+An _Org Owner_ is the administrator for a customer organization within the Cobalt app. As such,
+they can:
+
+- Add/remove the users of their choice, by their email addresses, as an _Org Member_ or a
+  _PT Team Member_.
```

Comment:
```suggestion
- Add/remove the users of their choice, by their email addresses, as an _Organization Member_ or a
  _Organization Owner_.
```

I realize we refer to "Org Owner" and "Org Member", but it's not clear to everyone, "What's an Org?"